Project.toml

```diff
@@ -5,11 +5,13 @@ version = "1.0.0-DEV"
 
 [deps]
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
the plotting library GLMakie should not be a dependency of thepackage. You could use CairoMakie in the docs - or have your own enviromnemt somewhere to add plotting libraries